test/classes/hannah/configAtomicsNotSupported.chpl

```diff
@@ -1 +1 @@
-config var n$ : atomic int;
+config var n : atomic int;
```

Comment:
Should there be a .compopts change with this?

Reply:
Ah, it's a config var good question.

I don't see a `.compopts` file for this test so I think not.